bin/dnsoterica

```diff
@@ -1,7 +1,6 @@
 #!/icg/bin/perl
 # ABSTRACT: run dns-o!
-package
-  dnsoterica;
+package dnsoterica;
```

Comment:
This new line is weird but intentional.  It prevents the PAUSE indexer from trying to treat this file as a package an indexing it.  If we eliminate the newline, an entry for `dnsoterica` will be made in CPAN's `02packages` index, which is not really useful.  So I'll probably revert this one change.

Admittedly, this is some pretty arcane stuff.